src/java.desktop/share/classes/javax/swing/plaf/nimbus/skin.laf

```diff
@@ -83,6 +83,9 @@
       <uiColor name="nimbusDisabledText">
          <matte red="142" green="143" blue="145" alpha="255" hueOffset="0.0" saturationOffset="0.0" brightnessOffset="0.0" alphaOffset="0"/>
       </uiColor>
+      <uiColor name="tabbedPaneContentArea">
```

Comment:
One more thing
All the above properties are prepended with nimbus except this so I guess it needs to be "nimbusTabbedPaneContentArea"

Reply:
There are properties without nimbus prepended as well... e.g. textInactiveText, activeCaption etc.
Anyways Updated with the suggested name.

Reply:
ok..In that case, this property should have been put after those and not in between nimbus prepended properties...
It's upto you to keep this or revert and move to appropriate location..

Reply:
Ok to keep this.